samples/05_content_publishers/updating_features_in_a_feature_layer.ipynb

```diff
@@ -1 +1 @@
nic  \\\n", "0   State        1  Census Designated Place   189029    25457     16229   \n", "1   State        2         City and Borough    15242      116      1040   \n", "2   State        3                     City    93773      302      8410   \n", "3   State        4                     City    22195      125      1863   \n", "4   State        5                     City    68172      643     19973   \n", "\n", "    latitude   longitude   males        name  objectid  pop2000  pop2007  \\\n", "0  21.305782 -157.823436  182628    Honolulu         1   371657   378587   \n", "1  58.351418 -134.511582   15469      Juneau         2    30711    31592   \n", "2  43.613736 -116.237655   92014  Boise City         3   185787   203529   \n", "3  47.042418 -122.893073   20319     Olympia         4    42514    45523   \n", "4  44.931109 -123.029155   68752       Salem         5   136924   152039   \n", "\n", "  state   white  \n", "0    HI   73093  \n", "1    AK   22969  \n", "2    ID  171204  \n", "3    WA   36246  \n", "4    OR  113746  "]}, "execution_count": 41, "metadata": {}, "output_type": "execute_result"}], "source": ["cities_fset3 = cities_flayer.query()\n", "cities_fset3.sdf.head(5)"]}, {"cell_type": "markdown", "metadata": {}, "source": ["## Conclusion\n", "In this sample, we observed an edit intensive method to keep feature layers updated. We published data from first spreadsheet as a feature layer. We then updated existing features from second spread sheet (used geometry module to project the coordinates in the process), and added new features. The third spreadsheet presented additional attribute columns which were added to the feature layer by editing its definition and then updating the features with this additional data.\n", "\n", "This method is editing intensive and you may choose this when the number of features to edit is less or if you needed to selectively update certain features as updates come in.\n", "\n", "An alternate method is to overwrite the feature layer altogether when you always have current information coming in. This method is explained in the sample [Overwriting feature layers](https://developers.arcgis.com/python/sample-notebooks/overwriting-feature-layers)"]}], "metadata": {"anaconda-cloud": {}, "esriNotebookRuntime": {"notebookRuntimeName": "ArcGIS Notebook Python 3 Standard", "notebookRuntimeVersion": "4.0"}, "kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.8.2"}, "toc": {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": true, "toc_position": {}, "toc_section_display": true, "toc_window_display": true}}, "nbformat": 4, "nbformat_minor": 1}
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    item_prop = {'title':'USA Capitals spreadsheet ' + now_ts}
</pre><p>Need <code>"item_type": "CSV"</code> for item properties as a required key value pair :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Oops and <code>.result()</code> to get the Folder.add() result!</p>